src/components/Plans/PlanCalculator/PlanPrice/PricePerMonth/index.js

```diff
@@ -18,7 +20,11 @@ export const PricePerMonth = ({ selectedPlan, discountPercentage }) => {
           {planFeeWithDiscount}
         </span>
       </h2>
-      <span className="dp-for-time">{_('plan_calculator.per_month')}</span>
+      {selectedPlan.type !== PLAN_TYPE.byCredit ? (
+        <span className="dp-for-time">{_('plan_calculator.per_month')}</span>
+      ) : (
+        ''
+      )}
```

Comment:
No se podría hacer algo más dependiente de una propiedad de los planes en lugar de atarlo al tipo de plan?

Reply:
Un ejemplo

Reply:
Ya lo hablamos, pero un ejemplo sería:

```suggestion
      {selectedPlan.isMontlhy ? (
        <span className="dp-for-time">{_('plan_calculator.per_month')}</span>
      ) : (
        ''
      )}
```

Ojo, tal vez no sea posible hacerlo.